requirements.txt

```diff
@@ -35,3 +35,4 @@ tsvalid
 j2cli[yaml]
```

Comment:
Rename file back to what it was, as the original name is needed to build ODK!

Reply:
FYI, the reason why this file is not named `requirements.txt` is given in commit 3fdd6bf1ada6c4e3ee9b2c7190e2660a69bd7732.

Reply:
Thank you, seems I forgot to restore the file name when development finished.And updated default output location to docs/ directory now.